docs/production-deployment/cloud/tcld/namespace.mdx

```diff
@@ -28,6 +30,57 @@ Alias: `n`
 - [tcld namespace retention](#retention)
 - [tcld namespace update-codec-server](#update-codec-server)
 
+## add-region
+
+Use `tcld namespace add-region` to add a standby region to an existing Temporal Cloud [Namespace](/namespaces).
```

Comment:
Should we link this to a Multi Region Namespaces section?

Reply:
We can't do that until MRN is pushed first because Vercel will reject the draft